### PR TITLE
Backend Kotlin

### PR DESCRIPTION
**Pull request by antoniofilhozup**
Creates a new directory for a backend written in Kotlin.

### CODE REVIEW

File: backend-kotlin/src/main/proto/payment.proto [hunk 1]
```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
```
Comment (sandokandias) [inline backend-kotlin/src/main/proto/payment.proto]: you could use the proto file that is in the proto folder of the repo and remove this file

**Review by sandokandias — CHANGES_REQUESTED** [review]

**Comment by sandokandias** [comment]
you need to add the directive in the Makefile to generate the kotlin stubs

File: backend-kotlin/src/main/proto/payment.proto (hunk 1 above)
Comment (antoniofilhozup) [reply backend-kotlin/src/main/proto/payment.proto]: Done, thanks @JoaoDanielRufino

**Comment by antoniofilhozup** [comment]
@sandokandias In order to generate the stub files, just build the application with the command `./gradlew build`.

**Review by sandokandias — APPROVED** [review]

**Pull request merged**